File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/dtos/menu/MenuSuggestedPortionResponseDTO.java
package com.robinfood.core.dtos.menu;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@AllArgsConstructor
@Data
public class MenuSuggestedPortionResponseDTO {

    private final List<MenuSuggestedPortionDataDTO> changes;
    private final Long id;
    private final String image;
    private final String name;
    private final Long parentId;
    private final String sku;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/dtos/menu/response/MenuSuggestedPortionResponseDTO.kt
package com.robinfood.core.dtos.menu.response

data class MenuSuggestedPortionResponseDTO(
        override var id: Long,
        override var image: String,
        override var name: String,
        override var parentId: Long,
        override var sku: String,
        val suggesteds: List<MenuSuggestedPortionDataDTO>
): MenuPortionDTO(id, image, name, parentId, sku)